tests/test_lingreedy.py

```diff
@@ -0,0 +1,647 @@
+import datetime
+import math
+from copy import deepcopy
+import numpy as np
+import pandas as pd
+from sklearn.preprocessing import StandardScaler
+
+from mabwiser.mab import LearningPolicy, NeighborhoodPolicy
+from tests.test_base import BaseTest
+
+
+class LinGreedyTest(BaseTest):
```

Comment:
Tests look good to me. Can you add one to confirm predictions for LinGreedy(epsilon=0, l2_lambda=X) == LinUCB(alpha=0, l2_lambda=X)?